src/vaults/BaseInvestmentManager.sol

```diff
@@ -72,6 +76,11 @@ abstract contract BaseInvestmentManager is Auth, Recoverable, IBaseInvestmentMan
         );
     }
 
+    /// @inheritdoc IBaseInvestmentManager
+    function poolEscrow(PoolId poolId) public view returns (IPoolEscrow) {
+        return poolEscrowProvider.escrow(poolId.raw());
+    }
```

Comment:
Nit: Since this is unused, let's remove it?

Reply:
I think this will be really valuable for the UI, right? Because they will easily know the managers? But can remove of course.